components/camel-nats/src/main/java/org/apache/camel/component/nats/NatsConfiguration.java

```diff
@@ -454,4 +460,55 @@ public boolean isTraceConnection() {
     public void setTraceConnection(boolean traceConnection) {
         this.traceConnection = traceConnection;
     }
+
+    /**
+     * Whether to enable JetStream support for this endpoint.
+     */
+    public boolean isJetstreamEnabled() {
+        return jetstreamEnabled;
+    }
+
+    /**
+     * Sets whether to enable JetStream support for this endpoint.
+     *
+     * @param jetstreamEnabled {@code true} to enable JetStream, {@code false} otherwise.
+     */
+    public void setJetstreamEnabled(boolean jetstreamEnabled) {
+        this.jetstreamEnabled = jetstreamEnabled;
+    }
+
+    /**
+     * The name of the JetStream stream to use. If not specified, a default stream
+     * will be used if JetStream is enabled.
+     */
+    public String getJetstreamName() {
+        return jetstreamName;
+    }
+
+    /**
+     * Sets the name of the JetStream stream to use.
+     *
+     * @param jetstreamName The name of the JetStream stream.
```

Comment:
Please remove the param, is not needed. We are going to generate documentation from the javadoc, this would mess the documentation page.